src/lib/isMobilePhone.js

```diff
@@ -36,6 +36,7 @@ const phones = {
   'en-US': /^((\+1|1)?( |-)?)?(\([2-9][0-9]{2}\)|[2-9][0-9]{2})( |-)?([2-9][0-9]{2}( |-)?[0-9]{4})$/,
   'en-ZA': /^(\+?27|0)\d{9}$/,
   'en-ZM': /^(\+?26)?09[567]\d{7}$/,
+  'es-CL': /^(\+?56|0)[2-9]\d{1}\d{7}$/,
```

Comment:
Looked like you changed the wrong place, should be on `en-KE`.

https://github.com/chriso/validator.js/blob/master/src/lib/isMobilePhone.js#L27

Reply:
Oh, scratch that, seen that you were doing for both Chile and KE, my bad.